FreeRTOS/Demo/RISC-V_Galois_P1/FreeRTOSConfig.h

```diff
@@ -100,7 +100,14 @@
 #define configTICK_RATE_HZ ((TickType_t)1000)
 #define configMAX_PRIORITIES (5)
 #define configMINIMAL_STACK_SIZE ((uint32_t)512) /* Can be as low as 60 but some of the demo tasks that use this constant require it to be higher. */
-#define configTOTAL_HEAP_SIZE ((size_t)(256 * 1024))
+#if defined(testgenOnFreeRTOS) && (__riscv_xlen == 64) && defined(useExtraStack)
+    #define configSTACK_DEPTH_TYPE uint32_t //the default ifndef is uint16_t

```

Comment:
Do you need to change to uint32_t because 64k is not enough for extra stack?

Reply:
I am using 64K exactly in one of the scripts. I can reduce it and retest if you hate it that much, but this gives me more flexibility.

Reply:
I was just curious. Indeed, 32-bit would be ideally the default, but that is out of scope for this PR.